S/Sundials/build_tarballs.jl

```diff
@@ -42,26 +43,7 @@ fi
 # We attempt to build for all defined platforms
 platforms = supported_platforms()
 
-products = [
-    LibraryProduct("libsundials_sunlinsolspfgmr", :libsundials_sunlinsolspfgmr),
-    LibraryProduct("libsundials_ida", :libsundials_ida),
-    LibraryProduct("libsundials_cvode", :libsundials_cvode),
-    LibraryProduct("libsundials_cvodes", :libsundials_cvodes),
-    LibraryProduct("libsundials_sunmatrixdense", :libsundials_sunmatrixdense),
-    LibraryProduct("libsundials_sunlinsolspbcgs", :libsundials_sunlinsolspbcgs),
-    LibraryProduct("libsundials_idas", :libsundials_idas),
-    LibraryProduct("libsundials_nvecserial", :libsundials_nvecserial),
-    LibraryProduct("libsundials_sunlinsoldense", :libsundials_sunlinsoldense),
-    LibraryProduct("libsundials_sunlinsolspgmr", :libsundials_sunlinsolspgmr),
-    LibraryProduct("libsundials_sunlinsolpcg", :libsundials_sunlinsolpcg),
-    LibraryProduct("libsundials_sunlinsolsptfqmr", :libsundials_sunlinsolsptfqmr),
-    LibraryProduct("libsundials_sunlinsolklu", :libsundials_sunlinsolklu),
-    LibraryProduct("libsundials_sunmatrixsparse", :libsundials_sunmatrixsparse),
-    LibraryProduct("libsundials_sunlinsolband", :libsundials_sunlinsolband),
-    LibraryProduct("libsundials_sunmatrixband", :libsundials_sunmatrixband),
-    LibraryProduct("libsundials_kinsol", :libsundials_kinsol),
-    LibraryProduct("libsundials_arkode", :libsundials_arkode),
-]
+products = Vector{LibraryProduct}()
```

Comment:
Revert this change?  Maybe there are more, I didn't check, but the old list of products seems to work for me.

Reply:
There should be 2 extra products for the LAPACK LinSolvers. I will run it locally with these changes to see if it works